newIDE/app/src/PropertiesEditor/PropertiesMapToSchema.js

```diff
@@ -220,7 +220,40 @@ const createField = (
       getLabel,
       getDescription,
     };
-  } else {
+  } else if(valueType ==='animationname')
+    {
+      function getChoices()
+    {
+      if(!object)
+      { return [{value:"Object is not valid !", label:"Object is not valid !"}] }
```

Comment:
The editor should ensure it never happens.
If we only allow to define "AnimationName" properties in the "Behavior properties" tab here (and not in "Scene properties":
![image](https://github.com/user-attachments/assets/099c544f-9d8d-4b86-b78e-0494f1d43319)

It should be fine to return an empty Array.